app/controllers/avo/search_controller.rb

```diff
@@ -60,14 +62,17 @@ def search_resource(resource)
 
       result_object = {
         header: header,
-        help: resource.fetch_search(:help) || "",
-        results: results,
+        help: resource.fetch_search(:help) || '',
+        results: results.map do |result|
+                   result.transform_values do |value|
+                     highlight(value.to_s, params[:q])
+                   end
```

Comment:
I meant to apply the highlight in `fetch_result_information` under the `_label` key.

This way we avoid doing another loop over the results.